components/dashboard/src/components/BillingAccountSelector.tsx

```diff
@@ -4,27 +4,25 @@
  * See License.AGPL.txt in the project root for license information.
  */
 
-import { useContext, useEffect, useState } from "react";
-import { Team, TeamMemberInfo } from "@gitpod/gitpod-protocol";
+import { Team } from "@gitpod/gitpod-protocol";
 import { AttributionId, AttributionTarget } from "@gitpod/gitpod-protocol/lib/attribution";
-import { getGitpodService } from "../service/service";
-import { useTeams } from "../teams/teams-context";
-import { UserContext } from "../user-context";
+import { useContext, useEffect, useState } from "react";
 import SelectableCardSolid from "../components/SelectableCardSolid";
+import { OrganizationInfo, useOrganizations } from "../data/organizations/orgs-query";
 import { ReactComponent as Spinner } from "../icons/Spinner.svg";
+import { getGitpodService } from "../service/service";
+import { UserContext } from "../user-context";
 import Alert from "./Alert";
-import { publicApiTeamMembersToProtocol, teamsService } from "../service/public-api";
 
 export function BillingAccountSelector(props: { onSelected?: () => void }) {
     const { user, setUser } = useContext(UserContext);
-    const teams = useTeams();
-    const [teamsAvailableForAttribution, setTeamsAvailableForAttribution] = useState<Team[] | undefined>();
-    const [membersByTeam, setMembersByTeam] = useState<Record<string, TeamMemberInfo[]>>({});
+    const orgs = useOrganizations();
+    const [teamsAvailableForAttribution, setTeamsAvailableForAttribution] = useState<OrganizationInfo[]>([]);
     const [errorMessage, setErrorMessage] = useState<string | undefined>();
 
     useEffect(() => {
-        if (!teams) {
-            setTeamsAvailableForAttribution(undefined);
+        if (orgs.isLoading) {
```

Comment:
Should we also be checking to make sure you have at least one org (if we're not loading anymore), or should that not be a possible scenario?

Reply:
This will be handled by the code below